cmd/git.go

```diff
@@ -5,13 +5,16 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/zricethezav/gitleaks/v8/cmd/scm"
+	"github.com/zricethezav/gitleaks/v8/detect"
 	"github.com/zricethezav/gitleaks/v8/logging"
 	"github.com/zricethezav/gitleaks/v8/report"
 	"github.com/zricethezav/gitleaks/v8/sources"
 )
 
 func init() {
 	rootCmd.AddCommand(gitCmd)
+	gitCmd.Flags().String("provider", "", "the SCM platform used to generate links (github, gitlab)")
```

Comment:
I'm undecided on what to call this flag. I don't love `--provider`.

Reply:
- `--scmp` for scm platform?
- `--git-host`

Reply:
I don't love any of these options either

Reply:
`--host` could work too because `git` is already present in the git scanning command:
`gitleaks git --host="github.com" -v .`

Reply:
I don't think `--host` makes sense for self-hosted instances (e.g., `--host="github.com"` when your company's URL is `github.company.org`).

`--platform` is probably better than `--provider`, at least.